applications/openshift/api-server/api_server_encryption_provider_cipher/rule.yml

```
@@ -76,7 +76,7 @@ ocil: |-
     API server to verify that its resources were successfully encrypted:
     <pre>
     # encrypt the etcd datastore
-    $ oc get openshiftapiserver -o=jsonpath='{range.items[0].status.conditions[?(@.type=="Encrypted")]}{.reason}{"\n"}{.message}{"\n"}'
+    $ oc get openshiftapiserver -o=jsonpath='{range .items[0]}{.status.conditions[?(@.type=="Encrypted")].status}{"\n"}{end}'
```

@prb112 Hi,

The jsonpath itself is not wrong, but it doesn't show anything meaningful when etcd encryption is not configured.

Soon after encryption is configured it outputs the following:
```
oc get openshiftapiserver -o=jsonpath='{range .items[0].status.conditions[?(@.type=="Encrypted")]}{.reason}{"\n"}{.message}{"\n"}'                                                                                                                                    
EncryptionInProgress
Resource routes.route.openshift.io is not encrypted


```

And when it is completed:
```
oc get openshiftapiserver -o=jsonpath='{range .items[0].status.conditions[?(@.type=="Encrypted")]}{.reason}{"\n"}{.message}{"\n"}'
EncryptionCompleted
All resources encrypted: routes.route.openshift.io


```

How about adding `.status` to the original jsonpath?
```suggestion
    $ oc get openshiftapiserver -o=jsonpath='{range .items[0].status.conditions[?(@.type=="Encrypted")]}{.status}{"\n"}{.reason}{"\n"}{.message}{"\n"}{end}
```

---

I made the change on [a4c0420](https://github.com/ComplianceAsCode/content/pull/11099/commits/a4c0420598969942ac9cb9b0cfebd168bc08ca28)